robottelo/ui/base.py

```diff
@@ -463,3 +420,24 @@ def is_element_visible(self, locator):
             return False
         self.wait_for_ajax()
         return element.is_displayed()
+
+    def click(self, locator, wait_for_ajax=True, timeout=30):
```

Comment:
Ok, sounds good. I just thought, that sometimes, we need to wait more than 12 sec for some elements and we should have ability to do that, but let's proceed further with refactor procedure...